src/main/resources/reference.conf

```diff
@@ -250,6 +250,11 @@ resourceTypes = {
           "read"
         ]
       }
+      # See https://broadworkbench.atlassian.net/browse/IA-5062
+      # This role can be removed after the migration is complete.
+      leo_migration = {
```

Comment:
can this be renamed `leo_migrator` to match the other role names?

Reply:
yes, good call

Reply:
Fixed